presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -60,7 +60,7 @@ statement
         (privilege (',' privilege)* | ALL PRIVILEGES)
         ON TABLE? qualifiedName FROM grantee=identifier                #revoke
     | SHOW GRANTS
-        ON (ALL | TABLE? qualifiedName)                                #showGrants
```

Comment:
That is the way the `GRANT` statement works in the spec https://ronsavage.github.io/SQL/sql-2003-2.bnf.html#grant statement